ionoscloud/provider.go

```diff
@@ -256,3 +256,12 @@ var resourceDefaultTimeouts = schema.ResourceTimeout{
 	Delete:  schema.DefaultTimeout(60 * time.Minute),
 	Default: schema.DefaultTimeout(60 * time.Minute),
 }
+
+// k8sTimeouts sets default value for each Timeout type
+// can be increased to a maximum of 4 hours
+var k8sTimeouts = schema.ResourceTimeout{
+	Create:  schema.DefaultTimeout(2 * time.Hour),
```

Comment:
Please extract the number 60 in a constant to be more clearly.
eg: resourceDefaultTimeoutsMinutes

Reply:
I'd argue that it just adds a level of indirection and that 60* time.Minute is pretty clear considering resourceDefaultTimeouts is the name of the variable.